src/chrome/content/rules/jobcloud.ch.xml

```diff
@@ -0,0 +1,13 @@
+<ruleset name="jobcloud.ch">
+	<target host="jobcloud.ch" />
+	<target host="www.jobcloud.ch" />
+	<target host="cdn.jobcloud.ch" />
+	<target host="confluence.jobcloud.ch" />
+	<target host="test.confluence.jobcloud.ch" />
+	<target host="img.jobcloud.ch" />
```

Comment:
Do you have a test URL for this? It's okay if you don't but it's better if you have one.